schemas/jsonschema/PosesInFrame.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$comment": "Generated from PosesInFrame by @foxglove/message-schemas",
+  "title": "PosesInFrame",
+  "description": "An array of timestamped poses in a named coordinate frame",
```

Comment:
Keep consistent with previous 2 descriptions, depending on what you decide those to be.